mods/tsunamissb/statuses.js

```diff
@@ -97,10 +97,10 @@ exports.BattleStatuses = {
 	tsunamiprince: {
 		exists: true,
 		onStart: function () {
-			this.add('c', '~Tsunami Prince', 'You think i came for the battle, when it\'s really your soul i want.');
+			this.add('c', '~Tsunami Prince', 'You think I came for the battle, when it\'s really your soul i want.');
```

Comment:
https://github.com/DesoGit/Tsunami_PS/pull/21
https://github.com/HoeenCoder/SpacialGaze/pull/312
 :P there corrected both in both places as I was already doing a PR for each today anyways. 
Kraken Mare, here to correct the world one word at a time. @DeathlyPlays